packages/hash/api/src/graphql/resolvers/user/util.ts

```diff
@@ -37,6 +37,41 @@ export const mapUserModelToGQL = async (
   };
 };
 
+type GQLOrgExternalResolvers =
+  | "emailInvitations"
+  | "invitationLinks"
+  | "memberships"
+  | "memberOf"
+  | "entityType"
+  | "linkGroups"
+  | "linkedEntities"
+  | "linkedAggregations";
+
+export type UnresolvedGQLOrg = Omit<GQLOrg, GQLOrgExternalResolvers>;
+
+export const mapOrgModelToGQL = async (
+  org: OrgModel,
+): Promise<UnresolvedGQLOrg> => {
+  return {
+    accountId: org.entityId,
+    id: org.entityId,
+    entityId: org.entityId,
```

Comment:
What is the difference between `id` and `entityId`?

Reply:
If I recall correctly, `id` was used by the GQL client for caching purposes. I'm not sure whether it's still needed now though given some of the changes to [the `createApolloClient`](https://github.com/hashintel/hash/blob/13e57f2d6b4a055b9b78023467b89b76906eb435/packages/hash/shared/src/graphql/createApolloClient.ts). It seems we're now explicitly telling the GQL client that `entityId` is the `keyField`. Perhaps a FE dev more familiar with the setup can confirm this? cc @kachkaev @nathggns

Reply:
Type policies make sense:
https://github.com/hashintel/hash/blob/13e57f2d6b4a055b9b78023467b89b76906eb435/packages/hash/shared/src/graphql/createApolloClient.ts#L83-L96

I used custom Apollo id fields in the past and the setup looked similar. Not 100% about syntax (can't recall details), but caching should be quite vivid with Apollo Dev Tools. Might be worth verifying it like that just in case.

https://www.apollographql.com/docs/react/development-testing/developer-tooling/#features

> Cache inspector: Visualize the Apollo Client cache and search it by field name and/or value.

Reply:
Thanks Alex, I think investigating this is beyond the scope of this PR. Have [created a task](https://app.asana.com/0/1202805690238892/1202980861294716/f) and added todos to the mapping functions in https://github.com/hashintel/hash/pull/1068/commits/f0558908f4d86be272103d4046e340f344e34cc1